change the redme file

imvisp opened the pull request:
-added some subpoints 
-added pictures.

vinzvinci reviewed (commented):
Hmm can you screenshot the work you did? Coz this will help us to know the results better and faster then we can look to the code after. And I suggest rename this pull request `title` to something related to what you did. Example: `docs: added mlh picture and subtopic` - just example. 

> By the way, thank you for contributing :)

imvisp commented:
> Hmm can you screenshot the work you did? Coz this will help us and the maintainer to know the results better and faster then we can look to the code after. And I suggest rename this pull request `title` to something related to what you did. Example: `docs: added mlh picture and subtopic` - just example.
> 
> > By the way, thank you for contributing :)

i made changes in pr you could see that.

muthuannamalai12 commented:
@imvisp Can we replace that s as Capital S in line 12

vinzvinci commented:
> > Hmm can you screenshot the work you did? Coz this will help us and the maintainer to know the results better and faster then we can look to the code after. And I suggest rename this pull request `title` to something related to what you did. Example: `docs: added mlh picture and subtopic` - just example.
> > > By the way, thank you for contributing :)
> 
> i made changes in pr you could see that.

I'm just suggesting, what I mean is provide more details before we go jump from it.

kcoder63 commented:
Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too

vinzvinci commented:
> Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too

- @muthuannamalai12 first. (discuss about lhd share and added picture). 

- Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture?

imvisp commented:
> > Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too
> 
> - @muthuannamalai12 first. (discuss about lhd share and added picture for the LHD). 
> 
> - Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture for the steps?

Yes

kcoder63 commented on README.md:
```diff
@@ -4,7 +4,13 @@
+<img align="center" src="main1.png" style="width:100%">
+<h3>Add your Email</h3>
+<img align="center" src="main2.png" style="width:100%">
+<h3>select this</h3>
```
I'd suggest Accept Discord rules and Code of Conduct rather than select this.

imvisp replied:
Ok will do this 👍

kcoder63 commented:
> > Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too
> 
> * @muthuannamalai12 first. (discuss about lhd share and added picture for the LHD).
> * Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture for the steps?

If I merge this, won't the changes @muthuannamalai12 made overridden? Since this PR is changes made on the older one

imvisp commented:
> > > Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too
> > 
> > * @muthuannamalai12 first. (discuss about lhd share and added picture for the LHD).
> > * Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture for the steps?
> 
> If I merge this, won't the changes @muthuannamalai12 made overridden? Since this PR is changes made on the olderAre you talking about my pr or muthu's ?

kcoder63 commented:
Yours, I merged muthu's PR first as suggested, now if I merged yours it might override his changes because you made changes on an older README

imvisp commented:
> Yours, I merged muthu's PR first as suggested, now if I merged yours it might override his changes because you made changes on an older README

So what I have to do now ?

kcoder63 commented:
Um yeah that should do

imvisp commented:
> Um yeah that should do

what about the old one can you delete ?

kcoder63 commented:
I'll close this and sorry for the trouble :(
